6tempcontrol

```diff
@@ -214,6 +221,12 @@ do
   GPU=0
   while (( GPU < GPUS ))
```

Comment:
What if we change its approach and use same as watchdog?
For example change the while loop to:
 for (( GPU=0; GPU < GPU_COUNT; GPU++))

Reply:
Would be the very same thing.

Reply:
But it's more eye catching ... ;)
I was thinking to unify variables and arguments and ... So both use same system. for example watchdog uses GPU_COUNT while temp control use GPUS.

Reply:
Yeah, uniforming names is always a good thing.